pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/core/realtime/SegmentCompletionManager.java

```diff
@@ -66,6 +66,7 @@
   private final Map<String, Long> _commitTimeMap = new ConcurrentHashMap<>();
   private final PinotLLCRealtimeSegmentManager _segmentManager;
   private final ControllerMetrics _controllerMetrics;
+  private final ControllerConf _controllerConf;
```

Comment:
u don't need to store the controller conf, if you are calling into PinotLLCRealtimeSegmentManager to deide whether splitcommit is configured.